templates/patterns/banners/banner_hero.ui_patterns.yml

```diff
@@ -1,5 +1,5 @@
 banner_hero:
-  label: "Banner: hero"
+  label: "Deprecated: Banner: hero"
```

Comment:
I don't understand why we deprecate this pattern instead of creating a new "Page banner" pattern. ECL defines the two types clearly so I think there is value in keeping them.